code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -222,6 +222,7 @@
 		data["beakerCurrentpH"] = null
 
 	var/chemicals[0]
+	var/recipes[0]
```

Comment:
You turned the recipes associative list, which used the keys of the recipes as the recipe names, into a numbered list, which still uses the keys (now numbers, instead of the recipe name) as its name. Good job.